pages/02.general/02.how-to-create-your-own-ad-filters/docs.en.md

```diff
@@ -998,9 +998,9 @@ This rule won't be optimized and will be available for Android only:
 Specify which platforms can apply this rule. List of existing platforms:
 
 
-* windows - Adguard for Windows (<https://filters.adtidy.org/windows/filters/2.txt>)
+* windows - Example: English filter for Windows - [https://filters.adtidy.org/windows/filters/2.txt] (https://filters.adtidy.org/windows/filters/2.txt)
 
-* mac -- Adguard for Mac (<https://filters.adtidy.org/mac/filters/2.txt>)
+* mac - Adguard for Mac (<https://filters.adtidy.org/mac/filters/2.txt>)
```

Comment:
Links are still broken, they lead to `https://filters.adtidy.org/mac/filters/2.txt> `